Makefile

```diff
@@ -40,7 +40,7 @@ install-dev-tools:  ## Installs all necessary cargo helpers
 	cargo install --locked dprint
 	cargo install cargo-llvm-cov
 	cargo install cargo-hack
-	cargo install cargo-udeps
+	cargo install cargo-udeps --locked
```

Comment:
```suggestion
	cargo install --locked cargo-udeps 
```
It doesn't matter but since we're installing dprint this way,,, we might as well do the same here.

Reply:
I just ordered other options as well